packages/EasyCore/tests/Doctrine/DBAL/Types/JsonbTypeTest.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Tests\Doctrine\DBAL\Types;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Types\ConversionException;
+use Doctrine\DBAL\Types\Type;
+use EonX\EasyCore\Doctrine\DBAL\Types\JsonbType;
+use EonX\EasyCore\Tests\AbstractTestCase;
+
+/**
+ * @covers \EonX\EasyCore\Doctrine\DBAL\Types\JsonbType
+ */
+final class JsonbTypeTest extends AbstractTestCase
+{
+    /**
+     * @return mixed[]
+     *
+     * @see testConvertToDatabaseValueSucceeds
+     * @see testConvertToPhpValueSucceeds
+     */
+    public function provideConvertToDatabaseValues(): array
+    {
+        return [
+            'null phpValue' => [
+                'phpValue' => null,
+                'postgresValue' => null,
+            ],
+            'empty phpValue' => [
+                'phpValue' => [],
+                'postgresValue' => '[]',
+            ],
+            'integer phpValue' => [
+                'phpValue' => 13,
+                'postgresValue' => '13',
+            ],
+            'float phpValue' => [
+                'phpValue' => 13.93,
+                'postgresValue' => '13.93',
+            ],
+            'string phpValue' => [
+                'phpValue' => 'a string value',
+                'postgresValue' => '"a string value"',
+            ],
+            'array og integers phpValue' => [
+                'phpValue' => [681, 1185, 1878, 1989],
+                'postgresValue' => '[681,1185,1878,1989]',
+            ],
+            'multidimensional array phpValue' => [
```

Comment:
How about an object?

Reply:
Done.  Added it `provideConvertToDatabaseValues` provider.